lib/sentry/context.ex

```diff
@@ -424,7 +424,7 @@ defmodule Sentry.Context do
       get_sentry_context()
       |> Map.update(@breadcrumbs_key, [map], fn breadcrumbs ->
         breadcrumbs = [map | breadcrumbs]
-        Enum.take(breadcrumbs, -1 * Sentry.Config.max_breadcrumbs())
+        Enum.take(breadcrumbs, 1 * Sentry.Config.max_breadcrumbs())
```

Comment:
Can you maybe rewrite this code to

```elixir
Enum.take([map | breadcrumbs], Sentry.Config.max_breadcrumbs())
```

?

Reply:
yeah haha what was I even thinking